websites/D/Dash Radio/presence.ts

```diff
@@ -0,0 +1,46 @@
+const presence = new Presence({
+  clientId: "761889098490183691"
+});
+
+let songName: HTMLElement,
+  songArtist: HTMLElement,
+  songNameS: string,
+  songArtistS: string;
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  };
+
+  presenceData.startTimestamp = Math.floor(Date.now() / 1000);
+      songName = document.querySelector(
+        "header.MuiAppBar-root > div.music-dataview-container > span.App-Player-Song-Title-Text"
+      );
+      if(songName == null) {
+        songNameS = "None";
+      } else if(songName != null) {
+        songNameS = songName.innerText;
+      }
+
+    songArtist = document.querySelector(
+      "header.MuiAppBar-root > div.music-dataview-container > span.App-Player-Song-Artist-Text"
+    );
+    if(songArtist == null) {
+      songArtistS = "None";
+    } else if(songArtist != null) {
+      songArtistS = songArtist.innerText;
+    }
+
+    presenceData.smallImageKey = "logo";
```

Comment:
Small image key is used for playing paused etc. icons
You can use our official ones from: https://github.com/PreMiD/Presences/tree/master/.resources